galaxy_ng/app/api/v3/serializers/sync.py

```diff
@@ -0,0 +1,48 @@
+from rest_framework import serializers
+from pulp_ansible.app.models import CollectionRemote
+from pulp_ansible.app.viewsets import CollectionRemoteSerializer
+from galaxy_ng.app.constants import COMMUNITY_DOMAINS, VALID_DISTRO_NAMES
+
+
+class SyncConfigSerializer(CollectionRemoteSerializer):
+    created_at = serializers.DateTimeField(source='pulp_created', required=False)
+    updated_at = serializers.DateTimeField(source='pulp_last_updated', required=False)
+    policy = serializers.ChoiceField(
+        choices=CollectionRemote.POLICY_CHOICES,
+        default=CollectionRemote.ON_DEMAND
+    )
+
+    class Meta:
+        model = CollectionRemote
+        fields = (
+            'name',
+            'url',
+            'auth_url',
+            'policy',
+            'requirements_file',
+            'created_at',
+            'updated_at',
```

Comment:
We still need to return the token, but as a write only field.

Here's an example showing how to do this: https://github.com/ansible/galaxy_ng/blob/master/galaxy_ng/app/api/ui/serializers/user.py#L35

Reply:
I had to re-define the fields adding `write_only=True` because `extra_vars` doesn't work when  we are overriding the `.validate()` method.